baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -22,16 +22,17 @@ This section uses the Augmented Backus-Naur Form (ABNF) notation of [[!RFC5234]]
 ## Definition
 
 ```ABNF
-list        =  list-member 0*179( OWS "," OWS list-member )
-list-member =  key OWS "=" OWS value *( OWS ";" OWS property )
-property    =  key OWS "=" OWS value
-property    =/ key OWS
-key         =  token ; as defined in RFC 2616, Section 2.2
-value       =  %x21 / %x23-2B / %x2D-3A / %x3C-5B / %x5D-7E
-               ; US-ASCII characters excluding CTLs,
-               ; whitespace, DQUOTE, comma, semicolon,
-               ; and backslash
-OWS         =  *( SP / HTAB ) ; optional white space, as defined in RFC 7230, Section 3.2.3
+list           =  list-member 0*179( OWS "," OWS list-member )
+list-member    =  key OWS "=" OWS value *( OWS ";" OWS property )
+property       =  key OWS "=" OWS value
+property       =/ key OWS
+key            =  token ; as defined in RFC 2616, Section 2.2
+value          =  *baggage-octet
```

Comment:
Should we consider capturing the maximum byte limit for the value as part of this grammar (similar to how it is done in the "list" definition)?

Reply:
We actually can't. There is no hard limit on the `value` but on the whole list-member. You can have a long key and a short value or a short key and a long value.

Reply:
Got it, thanks.